### PR TITLE
aria2c: how to specify destination filename

### PR DESCRIPTION
**Pull request by a-b**
Update example: how to specify the destination filename

- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/aria2c.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 - Download a URI to a file:
 
-`aria2c {{url}}`
+`aria2c -o {{filename}} {{url}}`
```
Comment (navarroaxel) [inline pages/common/aria2c.md:9]: we like the first example without arguments, could you add this as a new example? Maybe like this:

```suggestion
`aria2c {{url}}`

Download the contents of an URL to a file:

`aria2c -o {{filename}} {{url}}`
```

**Review by navarroaxel — COMMENTED** [review]
Hi @a-b, thanks for your contribution!

File: pages/common/aria2c.md (hunk 1 above)
Comment (a-b) [reply pages/common/aria2c.md:13]: I see where you're coming from LGTM

**Review by navarroaxel — APPROVED** [review]

**Review by mebeim — APPROVED** [review]
Thank you @a-b :)

**Pull request merged**